bedrock/firefox/templates/firefox/mobile/get-app.html

```diff
@@ -0,0 +1,62 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/sendto" %}
```

Comment:
We're losing the translations that have already been made for the mobile page :( Any way around that?

Reply:
I'm fairly certain the issue with using the widget are due to it being on demo, iirc from past testing whilst on demo.

@stephaniehobson - @hoosteeno indicated in the original issue that "We can copy the widget from /mobile entirely, and put in the center of the page on a white background"

If the strings in the page are intended to be verbatim to that used in /mobile, can simply point to that /firefox/mobile-2019.lang file?

Reply:
@stephaniehobson @ejregithub the ask for this page is en-US only right now, no translations required from another page. The SMS works via geo location, not page locale. On demo it defaults to assuming you’re in the US. I think we’re fine to move forward here.